packages/rollups-node/s6-overlay/s6-rc.d/stream-cleanup/script/up.sh

```diff
@@ -1,9 +1,8 @@
 #!/usr/bin/env sh
 
-if [ -z ${ENABLE_STREAM_CLEANUP+x} ];then
-    printf "stream-cleanup: ENABLE_STREAM_CLEANUP not set, skipping cleanup\n"
-else
+if [ ! -z ${ENABLE_STREAM_CLEANUP+x} ];then
     printf "stream-cleanup: ENABLE_STREAM_CLEANUP is set, cleaning redis streams...\n"
-    redis-cli -h ${REDIS_ENDPOINT} DEL chain-${CHAIN_ID}:dapp-${DAPP_CONTRACT_ADDRESS:2}:rollups-input;
     redis-cli -h ${REDIS_ENDPOINT} DEL chain-${CHAIN_ID}:dapp-${DAPP_CONTRACT_ADDRESS:2}:rollups-claims;
-fi
\ No newline at end of file
+    redis-cli -h ${REDIS_ENDPOINT} DEL chain-${CHAIN_ID}:dapp-${DAPP_CONTRACT_ADDRESS:2}:rollups-input;
```

Comment:
I know you didn't added this, but the stream name is wrong.

```suggestion
    redis-cli -h ${REDIS_ENDPOINT} DEL chain-${CHAIN_ID}:dapp-${DAPP_CONTRACT_ADDRESS:2}:rollups-inputs;
```